raw/sources.bib

```diff
@@ -1,9 +1,10 @@
 @article{Bodt2019b,
-  author      = {Aaley, Uday Raj and Bodt, Timotheus A.},
-  title       = {New Kusunda data: A list of 250 concepts},
-  journal     = {Computer-Assisted Language Comparison in Practice},
-  date        = {2020},
-  month = {apr},
-  day = {08},
-  url	= {https://calc.hypotheses.org/2414},
+	title        = {New Kusunda data: A list of 250 concepts},
+	author       = {Aaley, Uday Raj and Bodt, Timotheus A.},
+	year         = {2020},
+	journal      = {Computer-Assisted Language Comparison in Practice},
+	volume       = {3},
+	url          = {https://calc.hypotheses.org/2414},
+	date         = {2020-04-08}
```

Comment:
Number is missing:

```
number = {4},
```

Reply:
It's ok to criticize, although I was a bit confuse about the "wrong place". But all good now, I found the right file. So before I actually push the PR. The full bib is now: 
@article{Bodt2019b,
	title        = {New Kusunda data: A list of 250 concepts},
	author       = {Aaley, Uday Raj and Bodt, Timotheus A.},
	year         = {2020},
	journal      = {Computer-Assisted Language Comparison in Practice},
	volume       = {3},
        number   = {4},
	url          = {https://calc.hypotheses.org/2414},
	date         = {2020-04-08}. # keep it or remove
}